staticDHCPd/extensions/official/httpdb.py

```diff
@@ -88,74 +88,87 @@ def _parse_server_response(json_data):
 class _HTTPLogic(object):
     def __init__(self):
         from staticdhcpdlib import config
-        
+
         try:
             self._uri = config.X_HTTPDB_URI
         except AttributeError:
             raise AttributeError("X_HTTPDB_URI must be specified in conf.py")
         self._headers = getattr(config, 'X_HTTPDB_HEADERS', {})
-        self._post = getattr(config, 'X_HTTPDB_POST', True)
-        
+        self._post = getattr(config, 'X_HTTPDB_POST', False)
+        additional_info = getattr(config, 'X_HTTPDB_ADDITIONAL_INFO', {})
+        string_list = ['&%s=%s' % (key, value) for key, value in additional_info.iteritems()]
+        self._additional_info = ''.join(string_list)
+        self._name_servers = getattr(config, 'X_HTTPDB_DEFAULT_NAME_SERVERS', '')
+        self._lease_time = getattr(config, 'X_HTTPDB_DEFAULT_LEASE_TIME', 0)
+        self._serial = getattr(config, 'X_HTTPDB_DEFAULT_SERIAL', 0)
+
     def _lookupMAC(self, mac):
         """
         Performs the actual lookup operation; this is the first thing you should
         study when customising for your site.
         """
         global _parse_server_response
-        
         #If you need to generate per-request headers, add them here
         headers = self._headers.copy()
-        
+
         #You can usually ignore this if-block, though you could strip out whichever method you don't use
         if self._post:
-            data = json.dumps({
-             'mac': str(mac),
-            })
-            
+            data = json.dumps(dict(
+             {'mac': str(mac)},
+             **self._additional_info
+            ))
+
             headers.update({
              'Content-Length': str(len(data)),
              'Content-Type': 'application/json',
             })
-            
+
+            url = self._uri
```

Comment:
Im not sure that url is properly descriptive of this variable.  Its either the straight uri or its just the uri augmented with the mac and info.  I think since url is a pretty standard term this should be renamed to avoid confusion.